runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -55,31 +53,138 @@
 import net.runelite.http.api.worlds.WorldType;
 import net.runelite.http.api.xp.XpClient;
 
+import static com.google.common.base.MoreObjects.firstNonNull;
+import static net.runelite.client.plugins.xptracker.XpWorldType.NORMAL;
+
 @PluginDescriptor(
-	name = "XP Tracker"
+		name = "XP Tracker"
 )
 @Slf4j
 public class XpTrackerPlugin extends Plugin
 {
+	private final XpState xpState = new XpState();
+	private final XpClient xpClient = new XpClient();
 	@Inject
 	private PluginToolbar pluginToolbar;
-
 	@Inject
 	private Client client;
-
 	@Inject
 	private SkillIconManager skillIconManager;
-
 	private NavigationButton navButton;
 	private XpPanel xpPanel;
-
-	private final XpState xpState = new XpState();
-
 	private WorldResult worlds;
 	private XpWorldType lastWorldType;
 	private String lastUsername;
 
-	private final XpClient xpClient = new XpClient();
+	private static VarPlayer startGoalVarpForSkill(final Skill skill)
```

Comment:
Don't move large swaths of code for no reason

Reply:
I think this happened with auto-format in intelliJ, it puts static methods at the top of the class. Is this not convention?

Reply:
I've never seen auto format do that. It is convention, it just makes it hard to tell what has changed.